tcks/microprofile-config/pom.xml

```diff
@@ -0,0 +1,63 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>quarkus-build-parent</artifactId>
+        <groupId>io.quarkus</groupId>
+        <version>999-SNAPSHOT</version>
+        <relativePath>../../build-parent/pom.xml</relativePath>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>quarkus-tck-microprofile-config</artifactId>
+    <name>Quarkus - TCK - MicroProfile Config</name>
+
+    <properties>
+        <maven.deploy.skip>true</maven.deploy.skip>
+        <!-- TODO move the version to the build parent -->
+        <microprofile-config-api.version>1.3</microprofile-config-api.version>
```

Comment:
Well, let's do it maybe?

Reply:
+1 to moving MP versions alongside SmallRye ones in build parent

Reply:
Ok, I'll move it to build-parent but I believe this should be addressed globally (all versions from BOM) in a separate PR.